spyder/utils/stylesheet.py

```diff
@@ -183,16 +183,23 @@ def _customize_stylesheet(self):
             spacing='0px',
         )
 
-        # Remove margins around separators
+        # Remove margins around separators and decrease size a bit
         css['QMainWindow::separator:horizontal'].setValues(
             marginTop='0px',
-            marginBottom='0px'
+            marginBottom='0px',
+            # This is summed to the separator padding (2px)
+            width="3px",
+            # The default iamge is not visiblw at this size
```

Comment:
This comment has two typos and can also be written clearer.

```suggestion
            # Hide image because the default image is not visible at this size
```